fish-rust/src/builtins/mod.rs

```diff
@@ -1,2 +1,4 @@
 pub mod shared;
+
 pub mod wait;
+pub mod echo;
```

Comment:
should be sorted, I think rustfmt does that automatically.

Same goes for `try_get_rust_builtin` and the `RustBuiltin` enum definition.
They should be sorted alphabetically, like the existing C++ builtins are